src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -33,8 +34,9 @@ import {isEmptyObject} from '@src/types/utils/EmptyObject';
 import arraysEqual from '@src/utils/arraysEqual';
 import BaseSelectionListItemRenderer from './BaseSelectionListItemRenderer';
 import FocusAwareCellRendererComponent from './FocusAwareCellRendererComponent';
-import type {BaseSelectionListProps, ButtonOrCheckBoxRoles, FlattenedSectionsReturn, ListItem, SectionListDataType, SectionWithIndexOffset, SelectionListHandle} from './types';
+import type {ButtonOrCheckBoxRoles, FlattenedSectionsReturn, ListItem, SectionListDataType, SectionWithIndexOffset, SelectionListHandle, SelectionListProps} from './types';
 
+const DEFAULT_CONFIRM_BUTTON_CONTENT_PADDING = 80;
```

Comment:
Could you add a comment here explaining why this arbitrary value has been chosen (`80`)?

Reply:
I've actually removed the code for `shouldFooterContentStickToBottom`, since we're not using the prop anywhere anymore. This was needed to allow content flow below the footer button, which the Design team decided to not do.

`const DEFAULT_CONFIRM_BUTTON_CONTENT_PADDING = 80` was simply the extra content container padding needed, to offset the content by the footer button height.

The optimal solution to not have any magic number for this, would be to measure the footer content and set it dynamically, though that would mean a slight performance overhead and seemed unnecessary complicated to me.

Reply:
I do agree, the performance overhead isn't worth it.